<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEditConjuntoA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Incluir:     </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditIncluirConjuntoB"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonIncluirConjuntoB">
       <property name="text">
        <string>Incluir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CONJUNTO B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CONJUNTO A</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditTamanhoConjuntoB"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonCriarConjuntoB">
       <property name="text">
        <string>Criar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Tamanho:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Incluir:     </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditIncluirConjjuntoA"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonIncluirConjjuntoA">
       <property name="text">
        <string>Incluir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CONJUNTO</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Criar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tamanho:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditTamanhoConjuntoA"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CONJUNTO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditConjuntoB">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="labelResultado">
       <property name="text">
        <string>Resultado:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEditResultado"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>528</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonUniao">
         <property name="text">
          <string>UNIÃO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonIntersecao">
         <property name="text">
          <string>Intersecao</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDiferenca">
         <property name="text">
          <string>A - B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDiferencaBA">
         <property name="text">
          <string>B - A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>A C B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSubConjuntoBA">
         <property name="text">
          <string>B C A</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonDisjunto">
         <property name="text">
          <string>Disjunto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAmplitude">
         <property name="text">
          <string>Amplitude de A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAmplitudeB">
         <property name="text">
          <string>Amplitude de B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonProdutoEscalar">
         <property name="text">
          <string>Produto Escalar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonIdenticoAB">
         <property name="text">
          <string>IDENTICOS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButtonMaiorSubA">
         <property name="text">
          <string>Maior SubCadeia A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMaiorSubB">
         <property name="text">
          <string>Maior SubCadeia B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOrdenar">
         <property name="text">
          <string>Ordenação A e B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMediaA">
         <property name="text">
          <string>Media Aritimetica A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMediaB">
         <property name="text">
          <string>Media Aritimetica B</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
